xml/soc-operations-gpu-passthrough.xml

```diff
@@ -128,18 +117,182 @@ drwxr-xr-x 3 root root 0 Feb 19 16:09 /sys/kernel/iommu_groups/20/devices/0000:0
+    using whitelisting and the PCI alias for the device. Example:
+    </para>
+<screen>&prompt.root;cat /etc/nova/nova.conf.d/102-nova-pcipassthru.conf
+[pci]
+passthrough_whitelist = [{ "address": "0000:84:00.0" }]
+alias = { "vendor_id":"10de", "product_id":"1db4", "device_type":"type-PCI", "name":"a1" }
+
+</screen>
+    <para>
+    The example above shows how to configure a PCI alias a1 to request a PCI device with a <literal>vendor_id</literal>
+    of <literal>10de</literal> and a
+    <literal>product_id</literal> of <literal>1db4</literal>.
+    </para>
+    </step>
+    <step>
+    <para>Once the file is updated, restart nova compute on the compute node:</para>
+    <screen>&prompt.root;systemctl restart openstack-nova-compute
+    </screen>
+    </step>
+    </procedure>
+    </listitem>
+    <listitem>
+    <para>
+    <emphasis role="bold">Configure the Controller nodes</emphasis>
+    </para>
+    <procedure>
+    <step>
+    <para>
+    Follow the following steps for all the controller nodes. Login to the controller host and
+    change directory to <filename>/etc/nova/nova.conf.d</filename> and list all the files under that folder:
+    </para>
+<screen>&prompt.root;cd /etc/nova/nova.conf.d/
+ls -al
+</screen>
+    </step>
+    <step>
+    <para>Create a new nova configuration file under this folder and name the file appropriately in lexicographic
+    increasing order. For example: <filename>102-nova-pcipassthru.conf </filename>.
+    </para>
+    </step>
+    <step>
+    <para>
+    Add the following configuration entries to the file and the configuration entries for controller nodes
+    only specify PCI alias for the device from the compute host. For example:
+    </para>
+<screen>&prompt.root;cat /etc/nova/nova.conf.d/102-nova-pcipassthru.conf
+[pci]
+alias = { "vendor_id":"10de", "product_id":"1db4", "device_type":"type-PCI", "name":"a1" }
+</screen>
+    <para>
+    The example above shows how to configure a PCI alias a1 to request a PCI device with a <literal>vendor_id</literal>
+    of <literal>10de</literal> and a
+    <literal>product_id</literal> of <literal>1db4</literal>.
+    </para>
+    </step>
+    <step>
+    <para>Once the file is updated, restart nova api on the controller node:</para>
+<screen>&prompt.root;systemctl restart openstack-nova-api
+</screen>
+    </step>
+    </procedure>
+    </listitem>
+
+    <listitem><para>
+    <emphasis role="bold">Flavor Creation</emphasis>
+    </para>
+    <para>
+    Login into the controller node and create a new flavor or update an existing flavor with the property
+    <literal>"pci_passthrough:alias"</literal>. For example:
+    </para>
+<screen>&prompt.root;source .openrc
+openstack flavor create --ram 8192 --disk 100 --vcpu 8 gpuflavor
+openstack flavor set gpuflavor --property "pci_passthrough:alias"="a1:1"
+</screen>
+    <para>
+    In the property, <literal>"pci_passthrough:alias"="a1:1"</literal>, <literal>a1</literal> before the </literal>:</literal>
```

Comment:
it's failing on this one `before the </literal>:</literal>`

Reply:
Needs to be <literal>:</literal> - I probably did this, sorry!

Reply:
No problem. Fixed in the latest commit.